docs/references/kontena-yml.md

```diff
@@ -331,26 +366,12 @@ links:
   - mysql:wordpress-mysql
 ```
 
-With Kontena you can always reach services by their internal DNS (*service_name.grid.kontena.local*) and links are not needed for the service discovery.
-
+With Kontena you can always reach services by their internal DNS (*service_name.stack.grid.kontena.local*) and links are not needed for the service discovery. See [networking](#networking) for full details.
 
 Links also express dependency between services in the same way as `depends_on`, so they determine the order of service startup.
 
-#### net
-> **Note:** Version 1 only. In version 2 use [network_mode](#network_mode).
-
-Network mode.
-
-```
-net: "bridge"
-```
-
-```
-net: "host"
-```
 
 #### network_mode
```

Comment:
Move the description of `net: bridge` vs `net:host` from the deprecated `net` section to here?

BTW `net` is still supported, it works as an alias of `network_mode`.